podman.spec.rpkg

```diff
@@ -249,7 +263,15 @@ fi
 
 %files tests
 %license LICENSE
-%{_datadir}/%{name}/test
+%dir %{_datadir}/%{name}/test
+%dir %{_datadir}/%{name}/test/system
+%{_datadir}/%{name}/test/system/*
+
+%files tests-e2e
+%license LICENSE
+%dir %{_datadir}/%{name}/test
+%dir %{_datadir}/%{name}/test/e2e
+%{_datadir}/%{name}/test/e2e/*
```

Comment:
The integration tests depend on a lot more than that. They import code from all over the place in the podman repo and dependencies from `vendor/`. It also depend on tools from `hack/`. Plus you want `ginkgo` to run them, you may be able to get away without ginkgo but then you loose the ability to set cli options to configure things, e.g. parallel runs or flake retries.

This will very likely not work. IMO the only sensible approach to this is to ship the entire repo and use the makefile targets to run them. Everything else will just blow up eventually.

Reply:
hmmm, I think a `podman-src` or `podman-devel` subpackage that installs the entire source to `/usr/share/gocode/src/github.com/containers/podman` could solve this.

Reply:
How would that differ from fetching the SRPM + build deps?

Reply:
I mean, SRPMs are a Solved Problem. Creating a complicated new mechanism seems unnecessary?

Reply:
Looks like I can make do with the srpm. I'll update the container-selinux PR in a bit to run the e2e tests. Thanks @edsantiago